versions.yaml

```diff
@@ -75,7 +75,7 @@ assets:
       url: "https://github.com/cloud-hypervisor/cloud-hypervisor"
       uscan-url: >-
         https://github.com/cloud-hypervisor/cloud-hypervisor/tags.*/v?(\d\S+)\.tar\.gz
-      version: "v25.0"
+      version: "8f3462b3e356f56fee33ae4ba833a1fef012c1f0"
```

Comment:
You're tagging the commit as the release was not yet made, right?

Reply:
Right. It was a dry-run to make sure is working.